metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -472,26 +473,47 @@ class Compilers(
           }
 
           @tailrec
+          @nowarn
```

Comment:
Why do you need this?

Reply:
Otherwise we get warning that match is not exhaustive, but we can ignore since the length of this list will always be a multiple of 5